### ISSUE TITLE
Start run fails

### ISSUE THREAD

**Issue by XCarrel** [CPNV-ES/Runners-Laravel#93]
I have run that is ready both on the screen and in the app, it's 'my' run.
I start it but the call is rejected (403).

Digging a bit in the code turns up a weird piece of code in Run.ready():
```
    public function ready()
    {
        if ($this->status == 'published' || $this->status == 'finalizing') {
            return true;
        }
        return false;
    }
```

I would have expected 
```
    public function ready()
    {
        if ($this->status == 'ready') {
            return true;
        }
        return false;
    }
```

**Comment by bastiennicoud** [comment]
Indeed it is an oversight. I correct it quickly.

**Comment by bastiennicoud** [comment]
Fixed in hotfix c7ddf060d9535a591fc01e5df6e3d21cf8e12cb5
You can fetch master to get the fix.
